Header header

# Active Local View Cells
ActiveLocalViewCells lv_cells_

# Place Cell index (x,y,z) and rate
PlaceCellActivity pc_activity_


